### ISSUE TITLE
js-framework-benchmark

### ISSUE THREAD

**Issue by leeoniya** [justin-schroeder/arrow-js#16]
would be cool to see how arrow-js does in a `keyed` implementation of [krausest/js-framework-benchmark](https://github.com/krausest/js-framework-benchmark), and what that impl would look like.

**Comment by abishekatp** [comment]
Hi @justin-schroeder, Great work on this project. I have added the ArrowJs in [krausest/js-framework-benchmark](https://github.com/krausest/js-framework-benchmark). 
***the pull request is [here](https://github.com/krausest/js-framework-benchmark/pull/1253)***
I have tried it on my local machine and got the result which is in the attached image. Please let me know if there is a more optimized way to do this.
<img width="674" alt="Screenshot 2023-04-26 at 1 03 30 PM" src="https://user-images.githubusercontent.com/111952654/234506506-85100555-b4da-4927-ba4f-312430850b9f.png">

**Comment by justin-schroeder** [comment]
I've got a *much* improved version tuned for performance I'll be releasing before too long, including a `js-framework-benchmark` may or may not be worth keeping that PR open till then, but I'll plan to also submit one once that update has been made.

**Comment by abishekatp** [comment]
Yeah okay. May be if that PR got merged we can make the updated changes there itself or whatever way you prefer.
